core/templates/dev/head/services/ExplorationHtmlFormatterServiceSpec.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
year :)

Reply:
Done, changed it to 2017, or should it be 2018?